js/views/modals/purchase/Payment.js

```diff
@@ -1,12 +1,12 @@
 /*
-  This view is also used by the Order Detail overlay. If you make any changes, please
-  ensure they are compatible with both the Purchase and Order Detail flows.
-*/
+ This view is also used by the Order Detail overlay. If you make any changes, please
+ ensure they are compatible with both the Purchase and Order Detail flows.
+ */
```

Comment:
The changes in this view were copied from the multi-wallet order details branch.